.github/shared/build/action.yml

```
@@ -21,9 +21,9 @@ runs:
         key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}
     - name: Install dependencies
       shell: bash
-      run: yarn install --frozen-lockfile --non-interactive --logevel=error
+      run: yarn install --immutable
```

```suggestion
      run: yarn install --immutable --inline-builds
```

https://github.com/input-output-hk/cardano-js-sdk/blob/ab2265aa118b7f771055b5e3eab5621eea4f3476/.github/workflows/continuous-integration-unit-tests.yaml#L31

Maybe it's helpful to use this option too https://yarnpkg.com/cli/install#options

---

yes, thank you.